examples/react/multi-file-upload/src/style.css

```diff
@@ -0,0 +1,206 @@
+:root {
+  --w3ui-text-color: #fff;
+  --w3ui-overlay: rgba(29, 32, 39, 0.8);
+  --w3ui-gradient-stop-1: orange; 
+  --w3ui-gradient-stop-2: salmon;
+  --w3ui-button-color: #000;
+
+  --w3ui-uploader-container-padding: 0;
+  --w3ui-uploader-container-background: transparent;
+  --w3ui-uploader-padding: 2rem;
+  --w3ui-uploader-background: rgba(255, 255, 255, 0.05);
+  --w3ui-uploader-background-hover: rgba(255, 255, 255, 0.08);
+  --w3ui-uploader-width: 400px;
+  --w3ui-uploader-height: 200px;
+  --w3ui-uploader-border-color: rgba(255, 255, 255, 0.25);
+  --w3ui-uploader-border-radius: 8px;
+  --w3ui-uploader-primary: salmon;
+  --w3ui-uploader-primary-hover: rgb(250, 136, 124);
+}
+
+.w3ui-app {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
```

Comment:
I end up having to remove this on apps that scroll. maybe there is a way to be more flexible by default?

Reply:
Yes. ultimately we don't want this in here, but it makes it less ugly as we test these things out.